ladybug/dt.py

```diff
@@ -8,31 +8,39 @@
 class DateTime(datetime):
     """Create Ladybug Date time.
 
+    Args:
+        month: A value for month between 1-12 (Defualt: 1).
+        day: A value for day between 1-31 (Defualt: 1).
+        hour: A value for hour between 0-23 (Defualt: 0).
+        minute: A value for month between 0-59 (Defualt: 0).
+        leap_year: A boolean to indicate if datetime is for a leap year
+            (Default: False).
+
     Properties:
-        month
-        day
-        hour
-        minute
-        leap_year
-        doy
-        hoy
-        moy
-        int_hoy
-        float_hour
+        * month
+        * day
+        * hour
+        * leap_year
+        * doy
+        * hoy
+        * int_hoy
+        * max
+        * microsecond
```

Comment:
I know that, because the ladybug DateTime inherits from the native Python datetime , this `microsecond` property exists on the class. However, it is not one that is actually usable in any Ladybug workflows and, if users set this to be something other than 0, it can actually mess up certain workflows. So we would prefer to not list it here in the properties, even though I know we can't easily stop it from appearing in the docs. The same thing can be about the following properties, which should be removed from this list (as well as the `Date` and `Time` property lists):
```
* max
* min
* second
* resolution
```
Everything else can stay.

Reply:
Done!